src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImage.groovy

```diff
@@ -111,6 +115,10 @@ class DockerBuildImage extends AbstractDockerRemoteApiTask implements RegistryCr
             buildImageCmd.withPull(getPull())
         }
 
+        if(getLabels()) {
```

Comment:
Created [ISSUE](https://github.com/bmuschko/gradle-docker-plugin/issues/334) to implement checkstyle support so that users will be able to catch these locally and/or as part of CI.